### ISSUE TITLE
Error when trying to set attribute 'legendfontcolor' for backend gr

### ISSUE THREAD

**Issue by HDHarry** [JuliaPlots/Plots.jl#2261]
On the following setup:

```bash
Julia Version 1.2.0
Commit c6da87ff4b (2019-08-20 00:03 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin18.6.0)
  CPU: Intel(R) Core(TM) i5-8500 CPU @ 3.00GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
 ```

this very simple code breaks with an error:
```julia
using Plots
gr(legendfontcolor = :blue)
plot(rand(100), rand(100), legend = true)
``` 

The following error is produced: 
``` bash
MethodError: no method matching alpha(::Symbol)
Closest candidates are:
  alpha(!Matched::AGray32) at /Users/user/.julia/packages/ColorTypes/TvB32/src/traits.jl:11
  alpha(!Matched::ARGB32) at /Users/user/.julia/packages/ColorTypes/TvB32/src/traits.jl:10
  alpha(!Matched::RGB24) at /Users/user/.julia/packages/ColorTypes/TvB32/src/traits.jl:9
  ...
```

with the stack trace: 
```bash
Stacktrace:
 [1] gr_getcolorind(::Symbol) at /Users/user/.julia/packages/Plots/Iuc9S/src/backends/gr.jl:73
 [2] gr_set_textcolor(::Symbol) at /Users/user/.julia/packages/Plots/Iuc9S/src/backends/gr.jl:80
 [3] gr_display(::Plots.Subplot{Plots.GRBackend}, ::Measures.Length{:mm,Float64}, ::Measures.Length{:mm,Float64}, ::Array{Float64,1}) at /Users/user/.julia/packages/Plots/Iuc9S/src/backends/gr.jl:1548
 [4] gr_display(::Plots.Plot{Plots.GRBackend}, ::String) at /Users/user/.julia/packages/Plots/Iuc9S/src/backends/gr.jl:685
 [5] _show(::Base.GenericIOBuffer{Array{UInt8,1}}, ::MIME{Symbol("image/svg+xml")}, ::Plots.Plot{Plots.GRBackend}) at /Users/user/.julia/packages/Plots/Iuc9S/src/backends/gr.jl:1601
 [6] show(::Base.GenericIOBuffer{Array{UInt8,1}}, ::MIME{Symbol("image/svg+xml")}, ::Plots.Plot{Plots.GRBackend}) at /Users/user/.julia/packages/Plots/Iuc9S/src/output.jl:199
 [7] #sprint#342(::Nothing, ::Int64, ::typeof(sprint), ::Function, ::MIME{Symbol("image/svg+xml")}, ::Vararg{Any,N} where N) at ./strings/io.jl:107
 [8] sprint(::Function, ::MIME{Symbol("image/svg+xml")}, ::Vararg{Any,N} where N) at ./strings/io.jl:103
 [9] _ijulia_display_dict(::Plots.Plot{Plots.GRBackend}) at /Users/user/.julia/packages/Plots/Iuc9S/src/ijulia.jl:53
 [10] display_dict(::Plots.Plot{Plots.GRBackend}) at /Users/user/.julia/packages/Plots/Iuc9S/src/init.jl:42
 [11] #invokelatest#1 at ./essentials.jl:790 [inlined]
 [12] invokelatest at ./essentials.jl:789 [inlined]
 [13] execute_request(::ZMQ.Socket, ::IJulia.Msg) at /Users/user/.julia/packages/IJulia/F1GUo/src/execute_request.jl:112
 [14] #invokelatest#1 at ./essentials.jl:790 [inlined]
 [15] invokelatest at ./essentials.jl:789 [inlined]
 [16] eventloop(::ZMQ.Socket) at /Users/user/.julia/packages/IJulia/F1GUo/src/eventloop.jl:8
 [17] (::getfield(IJulia, Symbol("##15#18")))() at ./task.jl:268
```

In contrast, leaving the `legendfontcolor` option out works fine: 

```julia
using Plots

# works fine
gr()
plot(rand(100), rand(100), legend = true)
```

I'm kind of stuck here, because this particular error does not occur with other backends. 

While it may be something specific in the GR backend, I thought I leave this here because GR is the default backend. So any help would be appreciated.

**Comment by daschw** [comment]
Thanks, fixed in #2262, as workaround you can use `gr(legendfontcolor = plot_color(:blue))` until that is mergedd and released.

**Comment by HDHarry** [comment]
thanks for the help, that solved my problem
